src/erlcloud_sns.erl

```diff
@@ -422,6 +423,18 @@ publish_to_target(TargetArn, Message, Subject, Config) ->
 publish_to_target(TargetArn, Message, Subject, AccessKeyID, SecretAccessKey) ->
     publish_to_target(TargetArn, Message, Subject, new_config(AccessKeyID, SecretAccessKey)).
 
+-spec publish_to_phone(string(), sns_message()) -> string().
+publish_to_phone(TargetArn, Message) ->
+    publish_to_phone(TargetArn, Message, undefined, default_config()).
+
+-spec publish_to_phone(string(), sns_message(), aws_config()) -> string().
+publish_to_phone(TargetArn, Message, Config) ->
+    publish(phone, TargetArn, Message, undefined, Config).
+
+-spec publish_to_phone(string(), sns_message(), string(), string()) -> string().
+publish_to_phone(TargetArn, Message, AccessKeyID, SecretAccessKey) ->
```

Comment:
this conform rest of legacy code but under the hood it's not working since uses default region which is broken. just FYI.  that's fine to keep it this fun as default that need to be fixed separatelly.